### PR TITLE
localectl: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/localectl.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# localectl
+
+> Control the system locale and keyboard layout settings.
+> More information: <https://www.freedesktop.org/software/systemd/man/localectl.html>.
+
+- Show current settings of the system locale and keyboard mapping:
```
Comment (marchersimon) [inline pages/linux/localectl.md:6]:
```suggestion
- Show the current settings of the system locale and keyboard mapping:
```

File: pages/linux/localectl.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`localectl`
+
+- List available locales:
+
+`localectl list-locales`
+
+- Set a system locale variable:
+
+`localectl set-locale {{LANG}}={{en_US.UTF-8}}`
```
Comment (marchersimon) [inline pages/linux/localectl.md:16]:
```suggestion
`sudo localectl set-locale {{LANG}}={{en_US.UTF-8}}`
```

File: pages/linux/localectl.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Set the system keyboard mapping for the console and X11:
+
+`localectl set-keymap {{en}}`
```
Comment (marchersimon) [inline pages/linux/localectl.md:20]:
```suggestion
`sudo localectl set-keymap {{en}}`
```

File: pages/linux/localectl.md (hunk 2 above)
Comment (Managor) [reply pages/linux/localectl.md:16]: Is the sudo necessary since localectl will prompt you for the password anyways?

Comment (marchersimon) [reply pages/linux/localectl.md:16]: In may case it opens the GUI dialog, which is a bit annoying when you're working in the terminal, but that's probably just my setup. Generally, I think we should still include it, just to be safe.

Comment (Managor) [reply pages/linux/localectl.md:16]: Timedatectl exhibits the same behavior and it doesn't have sudo listed if we want to go by precedent. 
I personally would like to put as little as possible in the example command as long as it works. The simpler the better.

Comment (marchersimon) [reply pages/linux/localectl.md:16]: Sure, then let's leave it out.

**Review by marchersimon — APPROVED** [review]
Thanks @Managor

**Pull request merged**